README.md

```diff
@@ -4,79 +4,79 @@
 
 This release focus on the following:
 
-- We try to improve build performance with Persistent Caching.
-- We try to improve Long Term Caching with better algorithms and defaults.
-- We try to cleanup internal structures that were left in a weird state while implementing features in v4 without breaking changes.
-- We try to prepare for future features by doing breaking changes now to stay on v5 as long as possible.
+- We try and improve build performance with Persistent Caching.
+- We try and improve Long Term Caching with better algorithms and defaults.
+- We try and cleanup internal structures that were left in a weird state, while implementing features in v4 without introducing any breaking changes.
+- We try and prepare for future features by introducing breaking changes now, allowing us to stay on v5 for as long as possible.
 
 # Major Changes
 
-## Removed deprecated things
+## Removed Deprecated Items
 
-All things that were deprecated in v4 were removed.
+All items deprecated in v4 were removed.
 
-MIGRATION: Make sure that your webpack 4 build don't print deprecation warnings.
+MIGRATION: Make sure that your webpack 4 build does not print deprecation warnings.
 
-Here are a few things that were removed but have no deprecation warning in v4:
+Here are a few things that were removed but did not have deprecation warnings in v4:
 
-- IgnorePlugin and BannerPlugin must be passed an options object
+- IgnorePlugin and BannerPlugin must now be passed an options object.
 
-## Automatic node.js polyfills removed
+## Automatic Node.js Polyfills Removed
 
-In the early days webpack aimed to allow to run most node.js modules in the browser, but the module landscape changed and many modules uses are now written for frontend purposes. webpack <= 4 ships with polyfills for many of the node.js core modules. They are automatically applied once a module uses i. e. the `crypto` module.
+In the early days, Webpack's aim was to allow running most node.js modules in the browser, but the module landscape changed and many module uses are now written mainly for frontend purposes. webpack <= 4 ships with polyfills for many of the node.js core modules, which are automatically applied once a module uses any of the core modules (i.e. the `crypto` module).
 
-While this makes using modules written for node.js easy, it adds these huge polyfills to the bundle. In many cases these polyfills are even unneeded.
+While this makes using modules written for node.js easy, it adds these huge polyfills to the bundle. In many cases these polyfills are unnecessary.
 
-webpack 5 stopped to automatically polyfill these core modules and tried to focus on frontend-compatible modules.
+Webpack 5 stopped automatically polyfilling these core modules and focuses on frontend-compatible modules only.
```

Comment:
Webpack 5 stopped automatically polyfilling these core modules and focuses on frontend-compatible modules only
```suggestion
webpack 5 stops automatically polyfilling these core modules and focuses on frontend-compatible modules only.
```
Please read https://webpack.js.org/branding/#the-name for further info.